### PR TITLE
Feat : modal components

### PR DESCRIPTION
**Pull request by anuj-kumary**
- Added Modal Components
- Added link in the sidebar for modal and rating component

ScreenShot:-

Screenshot from 2022-03-22 19-00-12

### CODE REVIEW

File: components/Modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.modal {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    gap: 1rem;
+    width: 40rem;
```
Comment (samyakshah3008) [inline components/Modal/modal.css:6]: You can use max or min width for responsiveness, can refactor later on

File: components/Modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    box-shadow: rgb(100 100 111 / 20%) 0px 7px 29px 0px;
+    padding: 1rem 2rem;
```
Comment (samyakshah3008) [inline components/Modal/modal.css:8]: Good use of shorthand property

File: components/Modal/modal.html [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+      </ul>
+    </nav>
+
+    <div class="doc-container">
+      <aside class="sidebar">
+        <header class="sidebar__heading">Get Started</header>
+        <hr />
+        <ul>
+          <li class="sidebar__link">
+            <a href="/Documentation/docs.html">Introduction</a>
```
Comment (samyakshah3008) [inline components/Modal/modal.html:54]: Can put main content in `main` tag

**Review by samyakshah3008 — APPROVED** [review]
Given some minor suggestions and feedbacks as single comment
Rest is good for approval ✅

**Review by romabulani — APPROVED** [review]
- Nice use of semantic tags
- class names are quite understandable
- good to merge

**Pull request merged**